server/controllers/notification.js

```diff
@@ -0,0 +1,35 @@
+const Notification = require("../models/Notification");
+const asyncHandler = require("express-async-handler");
+
+exports.createNotification = asyncHandler(async (req, res) => {
```

Comment:
```suggestion
exports.createNotification = asyncHandler(async (req, res) => {
    const { type, description, receivedBy } = JSON.parse(req.body)
    const userId = req.user.id
    
    const profile = Profile.findById(userId);
    
    if (!profile) {
        return res.status(404).send("No Profile Found")
    }
    
    const newNotifcation = await Notification.create({
        type,
        description,
        receivedBy,
        createdBy: profile._id
    })
    
    return res.status(201).json({
        success: {
            notification: newNotification
        }
    })
    
});
```
I must've missed this the first time but this controller looks off

Reply:
Hey @rajivtitus, I implemented the change as recommended in commit https://github.com/hatchways/team-wintergreen/commit/8dc5de0100f881ad775d8659cc2c07cc60c7f505 on branch notifications-integration which is based on this branch.